lib/chef/resource/registry_key.rb

```diff
@@ -150,6 +164,10 @@ class RegistryKey < Chef::Resource
       }
       property :recursive, [TrueClass, FalseClass], default: false
       property :architecture, Symbol, default: :machine, equal_to: %i{machine x86_64 i386}
+      property :only_record_changes, [TrueClass, FalseClass],
+               default: false,
+               introduced: "19.0",
+               description: "Suppress reporting of the current value of sibling values in a registry key."
```

Comment:
suggestion: "Suppress reporting the sibling values of the current registry key."